multicluster/apis/multicluster/v1alpha1/multiclusterconfig_types.go

```diff
@@ -57,6 +57,9 @@ type MultiClusterConfig struct {
 	// ClusterSet and allow Antrea-native policies to select peers from other clusters
 	// in a ClusterSet.
 	EnableStretchedNetworkPolicy bool `json:"enableStretchedNetworkPolicy,omitempty"`
+	// Enable EndpointSlice to watch EndpointSlice changes only for exported Service, this is required
+	// when EndpointIPType is PodIP and CNI's EndpointSlice feature is enabled.
+	EnableEndpointSlice bool `json:"enableEndpointSlice,omitempty"`
```

Comment:
I wonder why a configuration parameter is needed but not use the feature gate like AntreaProxy? Ideally EndpointSlice should be the default API we use to get Endpoints as it's GA and more efficient. User shouldn't care whether we use which API we are using.

In AntreaProxy, we use EndpointSlice when the feature gate is enabled (the current default value) and will fall back to Endpoint API if the EndpointSlice API is not available (K8s <1.21)

Reply:
Make sense, we didn't introduce feature gate in MC, but I guess it's better to follow AntreaProxy way to check EndpointSlice API and fall back to Endpoint if it's not available. I will refine this part. thanks.

Reply:
I addressed the comment in a new commit, since Antrea MC didn't provide feature gate for EndpointSlice, the current behavior is to use EndpointSlice by default as long as the API is available and fall back to Endpoint when it's not available. @jianjuns any thoughts?

Reply:
Do we have a way to know if Endpoints are also created? In the current implementation, watching/exporting Endpoints is more efficient than exporting EndpointSlice.

Reply:
EndpointSlice will be created automatically if Endpoint is created, but I doubt Endpoint controller will do the same if EndpointSlice is created for a Service. @tnqn any idea?

Reply:
Since Endpoints will be mirrored to EndpointSlice and exporting Exporting Endpoints is more efficient in Multicluster, why do we need this new patch? I guess there is no EndpointSlice -> Endpoints mirroring, but you can test it by creating an EndpointSlice.

Reply:
I tested there is automatic Endpoints -> EndpointSlice mirroring and no EndpointSlice -> Endpoints mirroring. And the official doc about Services without selector guides users to create EndpointSlice manually now. If we want to support Services without selector, the best choice seems listening to EndpointSlice API when it's available because:
1. If user create EndpointSlice manually, it can be exported.
2. If user create Endpoints manually, it will be mirrored to EndpointSlice and be exported.

Reply:
Ok. Got it.